src/backend/index.js

```diff
@@ -1,21 +1,36 @@
 // @ts-check
-import os from 'os';
+// import os from 'os';
 import debug from 'debug';
-import packageJson from './package.json';
 import rn_bridge from 'rn-bridge';
+import {createServer} from 'rpc-reflector';
+import {MapeoClient} from './mapeo-core/index';
+import MessagePortLike from './message-port-like.js';
+import {mockData} from './mockData.js';
 
 // TODO: Account for args passed from node.startWithArgs
 debug.enable('*');
 
-const log = debug('mapeo-mobile-node-next');
+// @ts-expect-error
+const channel = new MessagePortLike(rn_bridge.channel);
+channel.start();
```

Comment:
Can you move this (`.start()`) to after the RPC server is created? Strictly this will be ok as-is because everything happens in the same tick, but if someone changes that, e.g. with an await before `createServer()`, then there would be a chance that messages would be lost.